completions/bash/podman

```diff
@@ -3345,6 +3345,18 @@ _podman_pod_unpause() {
     esac
 }
 
+__podman_pod_inspect() {
```

Comment:
__podman_pod_inspect() can this starts with single `_`?

Reply:
Not only can, but I think should.  Nice spot @QiWang19 , I missed that.

Reply:
DONE :+1: